tests/integration/SdkTest_test.cpp

```diff
@@ -2272,14 +2369,17 @@ TEST_F(SdkTest, SdkTestShares)
     string oldLink = link;
     link = "";
     nfile1 = std::unique_ptr<MegaNode>{megaApi[0]->getNodeByHandle(hfile1)};
-    ASSERT_NO_FATAL_FAILURE( createPublicLink(0, nfile1.get()) );
+    ASSERT_NO_FATAL_FAILURE( createPublicLink(0, nfile1.get(), 0, maxTimeout, mApi[0].accountDetails->getProLevel() == 0) );
     ASSERT_STREQ(oldLink.c_str(), link.c_str()) << "Wrong public link after link update";
 
 
     // Try to update the expiration time of an existing link (only for PRO accounts are allowed, otherwise -11
-    ASSERT_NO_FATAL_FAILURE( createPublicLink(0, nfile1.get(), 1577836800) );     // Wed, 01 Jan 2020 00:00:00 GMT
+    ASSERT_NO_FATAL_FAILURE( createPublicLink(0, nfile1.get(), m_time() + 30*86400, maxTimeout, mApi[0].accountDetails->getProLevel() == 0) );    
```

Comment:
Good spotting! I guess I thought that 2020 was too far away, back in 2016 :smile: (https://github.com/meganz/sdk/commit/664d7aaa385c8b63c72c7e6c6d648599c435296a#diff-ca1c9e26e592f97aedec977e06d12cd2R1525)

Reply:
I think this one was causing the test to fail, otherwise I would not have noticed - should be fine now though